lldb/source/Target/Platform.cpp

```diff
@@ -1197,6 +1197,34 @@ Status Platform::PutFile(const FileSpec &source, const FileSpec &destination,
   if (!source_file)
     return Status(source_file.takeError());
   Status error;
+
+  bool requires_upload = true;
+  {
```

Comment:
Why is this all in its own block?

Reply:
This is how I write code to scope the variables `dest_md5_low`, `dest_md5_high`, `success` away from the main scope of the function.

Is this not good to do in the llvm-project?

Reply:
If you really need a new scope, a small static function in the same file is the usual way. In this case it's fine to have them in the scope of the main function I think, I don't think they overlap with any existing var.

Reply:
I see. I wouldn't say it's a bad thing to create a new scope if you want to contain the lifetime of these variables to their own scope, but in LLDB (and more broadly LLVM) this is mostly used for RAII. The most common instance of this is when one wants to acquire a lock but does not want the lock to span the lifetime of the method body.

For consistency I would say either put this code in the main body, put it in a small static function, or put it in a lambda.

Reply:
Just removed the scoping - so the md5 vars are part of the main scope.